apps/app-frontend/src/components/ui/instance_settings/WindowSettings.vue

```diff
@@ -111,17 +111,7 @@ const messages = defineMessages({
           {{ formatMessage(messages.fullscreenDescription) }}
         </p>
       </div>
-      <Toggle
-        id="fullscreen"
-        :model-value="overrideWindowSettings ? fullscreenSetting : globalSettings.force_fullscreen"
```

Comment:
I think this gets rid of the behavior of it displaying the global setting if overriding is disabled